Explorer/Assets/Scripts/ECS/Abstract/BaseSystemsGroup.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ECS.Abstract
+{
+    /// <summary>
+    /// Simple group without arguments and with a Single Update
+    /// </summary>
+    public class BaseSystemsGroup : ISystem
```

Comment:
What is the difference between Arch.System.Group

Reply:
Contains no arguments, it is an outline for the future, currently not used, I just had some ideas to try